### PR TITLE
Add a class for the active navigation dot

### PR DESCRIPTION
**Pull request by nelsonwong1012**
This is a solution for https://github.com/elrumordelaluz/reactour/issues/103.

Added the class name `reactour__dot--active`. I tried to follow your class naming convention but am happy to change it to whatever you prefer.

### CODE REVIEW

File: src/Tour.js [hunk 1]
```diff
@@ -517,6 +517,7 @@ class Tour extends Component {
                         disabled={current === i || disableDotsNavigation}
                         showNumber={showNavigationNumber}
                         data-tour-elem="dot"
+                        className={current === i ? 'reactour__dot--active' : ''}
```
Comment (elrumordelaluz) [inline src/Tour.js]: I think it's better to position the contant `className` with the other ones [here](https://github.com/elrumordelaluz/reactour/blob/27b5dfc35767ead3732d883fe7cfb1ceb35a047d/src/Tour.js#L563-L573) using it something like:

```js
<Dot
  {...props}
  className={cn(CN.dot.base, {
    [CN.dot.active]: current === i,
  })}
/>
```

**Comment by elrumordelaluz** [comment]
Awesome @nelsonwong1012 !

just a little change to stay aligned with the other `className`s

File: src/Tour.js [hunk 2]
```diff
@@ -569,6 +572,10 @@ const CN = {
     base: 'reactour__helper',
     isOpen: 'reactour__helper--is-open',
   },
+  dot: {
+    base: 'reactour__dot',
+    active: 'reactour__dot--active',
```
Comment (nelsonwong1012) [inline src/Tour.js]: @elrumordelaluz 

I implemented your suggestions. Sorry, I wasn't familiar enough with your codebase.

Also, wondering if we really need a `base` class here. Or if we should call this `reactour__dot--is-active` to be more consistent with the naming?

Comment (elrumordelaluz) [reply src/Tour.js]: no problem at all @nelsonwong1012 
The _base_ `class` could be useful at the moment of customizing, like your needs for the _active_ one but for the _inactive_ ones.

LGFM `reactour__dot--is-active`

**Pull request merged**